test/data/error-custom/schema.js

```diff
@@ -0,0 +1,7 @@
+const gql = require('graphql-tag');
+
+module.exports = gql`
+  type Query {
+    customError: String
```

Comment:
Are we gonna request errors in every query we make ?

Reply:
No. It's just a test case to verify that "custom" errors (i.e. with an `extensions` field) are returned properly if they are thrown by the remote schema (`delegatetoSchema`).

Reply:
Cool. I get it !!